modules/data-streams/src/yamlRestTest/resources/rest-api-spec/test/data_stream/200_logs_datastream_defaults.yml

```
@@ -0,0 +1,111 @@
+---
+Verify default logs-*-* settings and mappings:
+  - do:
+      indices.create_data_stream:
+        name: logs-test-1
+  - is_true: acknowledged
+
+  - do:
+      indices.get_data_stream:
+        name: logs-test-1
+  - set: { data_streams.0.indices.0.index_name: idx0name }
+
+  # default backing index settings should be "ignore_malformed": true
+  - do:
+      indices.get_settings:
+        index: $idx0name
+  - match: { .$idx0name.settings.index.mapping.ignore_malformed: "true" }
+
+  # add test field mapping
+  - do:
+      indices.put_mapping:
+        index: $idx0name
+        body:
+          properties:
+            numeric_field:
+              type: integer
+  - is_true: acknowledged
+
+  # default backing index mapping should contain an exception for the @timestamp field - "ignore_malformed": false
+  - do:
+      indices.get_mapping:
+        index: $idx0name
+  - match: { .$idx0name.mappings.properties.@timestamp.ignore_malformed: false }
+  - match: { .$idx0name.mappings.properties.numeric_field.type: "integer" }
+
+  - do:
+      index:
+        index: logs-test-1
+        refresh: true
+        body:
+          '@timestamp': '2023-04-18'
+          message: 'valid'
+          numeric_field: 42
+  - match: {result: "created"}
+
+  - do:
+      index:
+        index:  logs-test-1
+        refresh: true
+        body:
+          '@timestamp': '2023-04-18'
+          message: 'invalid_type_valid_value'
```

nit: potentially more descriptive

```suggestion
          message: 'number_as_string'
```